deploy/charts/rook-ceph-cluster/values.yaml

```diff
@@ -361,17 +361,17 @@ cephClusterSpec:
     # deviceFilter:
     # config:
     #   crushRoot: "custom-root" # specify a non-default root label for the CRUSH map
-    #   metadataDevice: "md0" # specify a non-rotational storage so ceph-volume will use it as block db device of bluestore.
+    #   metadataDevice: "nvme0n1" # specify a non-rotational storage so ceph-volume will use it as block db device of bluestore.
     #   databaseSizeMB: "1024" # uncomment if the disks are smaller than 100 GB
     #   osdsPerDevice: "1" # this value can be overridden at the node or device level
     #   encryptedDevice: "true" # the default value for this option is "false"
+    # nodes:
     # # Individual nodes and their config can be specified as well, but 'useAllNodes' above must be set to false. Then, only the named
     # # nodes below will be used as storage resources. Each node's 'name' field should match their 'kubernetes.io/hostname' label.
-    # nodes:
     #   - name: "172.17.4.201"
     #     devices: # specific devices to use for storage can be specified for each node
     #       - name: "sdb"
-    #       - name: "nvme01" # multiple osds can be created on high performance devices
+    #       - name: "nvme1n1" # multiple osds can be created on high performance devices
```

Comment:
This is a good catch. @Underknowledge How about focusing on this change (fixing a type) in this PR and changing both the commit message and the PR description?